<ui version="4.0" >
 <class>DlgUsers</class>
 <widget class="QDialog" name="DlgUsers" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>312</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Benutzerverwaltung</string>
  </property>
  <property name="windowIcon" >
   <iconset resource="juliusd.qrc" >:/images/juliustray.png</iconset>
  </property>
  <layout class="QGridLayout" >
   <item row="0" column="0" >
    <widget class="QLabel" name="lbDbUsers" >
     <property name="text" >
      <string>Benutzer in der Datenbank:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2" >
    <widget class="QTableView" name="tvUsers" />
   </item>
   <item row="2" column="0" >
    <widget class="QPushButton" name="pbAddUser" >
     <property name="text" >
      <string>Benutzer hinzufügen</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1" >
    <widget class="QPushButton" name="pbDeleteUser" >
     <property name="text" >
      <string>Ausgewählten Benutzer löschen</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="juliusd.qrc" />
 </resources>
 <connections/>
</ui>
